README.md

```diff
@@ -1 +1,170 @@
-# testing-to-do-list
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 To Do List ](#-to-do-list-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+> This project is related to daily to dos.
+
+**To Do List** is a HTML and JavaScript based project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML, CSS, JavaScript and Webpack
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **You can manage your daily tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://maleemh.github.io/to-do-list/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Local Machine, Git, Text Editor (VS Code preferably)
+
+### Setup
+
+Create a folder to clone github repository.
+
+### Install
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/MAleemH/to-do-list.git
+```
+
+### Usage
+
+Open your terminal of VS Code and write following commands to run this project on local server:
+
+```sh
+  npm run build
+  npm run start
+```
+
+### Run tests
+
+After the above step, program will start running. You can perform desired activities.
+
```

Comment:
- [ ] Explainto potential users the command they need to write to run tests 
![image](https://user-images.githubusercontent.com/62705947/215113003-8a3a1c6a-9c40-4e4f-9e17-be1451136240.png)